src/internal.c

```diff
@@ -619,11 +622,27 @@ static int ExportCipherSpecState(WOLFSSL* ssl, byte* exp, word32 len, byte ver)
     exp[idx++] = specs->pad_size;
     exp[idx++] = specs->static_ecdh;
 
-    if (idx != DTLS_EXPORT_SPC_SZ) {
-        WOLFSSL_MSG("DTLS_EXPORT_SPC_SZ needs updated and export version");
+    if (idx != WOLFSSL_EXPORT_SPC_SZ) {
+        WOLFSSL_MSG("WOLFSSL_EXPORT_SPC_SZ needs updated and export version");
```

Comment:
Should `WOLFSSL_EXPORT_SPC_SZ` be an overridable macro? Currently it is a fixed enum value.